### PR TITLE
feat(starfish): add vcd measurements

### PR DESCRIPTION
**Pull request by DominikB2014**
Adds a VCD measurement to the following components

- WSV service breakdown chart
- Endpoint view breakdown chart
- Span summary view transaction list
- Span summary panel sample list

### CODE REVIEW

File: static/app/views/starfish/views/spanSummaryPage/spanTransactionsTable.tsx [hunk 1]
```diff
@@ -112,17 +113,22 @@ export function SpanTransactionsTable({
...
-        columnOrder={COLUMN_ORDER}
-        columnSortBy={[]}
-        grid={{
-          renderHeadCell: col => renderHeadCell({column: col, sort, location}),
-          renderBodyCell,
-        }}
-        location={location}
-      />
+      <VisuallyCompleteWithData
+        id="SpanSummary.EndpointTable"
```
Comment (gggritso) [inline static/app/views/starfish/views/spanSummaryPage/spanTransactionsTable.tsx:117]: I think this should be 
```suggestion
        id="SpanSummary.SpanTransactionsTable"
```

to match the component name

File: static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx [hunk 2]
```diff
@@ -110,42 +113,51 @@ export function SpanGroupBreakdown({
...
-              organization,
-              selected: Object.keys(event.selected).filter(key => event.selected[key]),
-              toggled: event.name,
-            });
-          }}
-        />
+        <VisuallyCompleteWithData
+          id={`WSV.${isEndpointBreakdownView ? 'EndpointBreakdown' : 'ServiceBreakdown'}`}
+          hasData={data.length > 0}
+        >
```
Comment (gggritso) [inline static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx:119]: I think it'd be okay to just call this `SpanGroupBreakdown`! You don't need the conditional IMO, it'd be fine to just have two separate widgets in a dashboard, each one filtering by the relevant transactions

File: static/app/views/starfish/views/spans/spansTable.tsx [hunk 3]
```diff
@@ -97,12 +97,15 @@ export default function SpansTable({
     });
   };
 
+  const isEndpointOverview = Boolean(endpoint);
```
Comment (gggritso) [inline static/app/views/starfish/views/spans/spansTable.tsx:100]: Definitely fine, but maybe hides the intent a bit. Having a `shouldTrackVCD` prop might be better, then it's clearer that this component only tracks its VCD on certain pages

**Review by gggritso — COMMENTED** [review]
LGTM, nits only!

File: static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/starfish/views/webServiceView/spanGroupBreakdown.tsx:119]: Yeah that's true! I think I coded this up before considering the dashboard, hence the conditional. I'll make this change

File: static/app/views/starfish/views/spans/spansTable.tsx (hunk 3 above)
Comment (DominikB2014) [reply static/app/views/starfish/views/spans/spansTable.tsx]: good catch, i like that change

**Review by gggritso — APPROVED** [review]

**Pull request merged**